joplin/api/schema.py

```diff
@@ -707,26 +707,17 @@ class Meta:
         interfaces = [graphene.Node, DepartmentResolver]
 
     def resolve_document(self, info):
+        document = self.get_document
```

Comment:
Moved all this logic into the OfficialDocumentPage model, since we'll need this for our site_search view as well.